documentation/developer_updates.md

```diff
@@ -5,20 +5,171 @@ we're making.
 
 ## November 2020
 
-### Puppet 7 Release Prep
+### Changes coming in Bolt 3.0
+
+With the new year fast approaching, the team is preparing to release the next
+major version of Bolt: **Bolt 3.0**! Over the next few releases, you will start
+to see deprecation warnings for specific features and configuration options
+that are slated for removal in Bolt 3.0. To help you prepare for the release,
+we've compiled a list of expected changes and removals.
+
+#### Changes
+
+- **Ship with Puppet 7**
+
+  Bolt packages will ship with Puppet 7. This change should not impact most
+  users, but changes to the plan language in Puppet may require updating your
```

Comment:
```suggestion
  users, but changes to the Puppet language may require updating your Bolt
```

I think this is also technically true for yaml plans, since some changes will impact resource steps.